tools/EMLassemblyline/geo_cov_template.xml

```diff
@@ -1,6 +1,6 @@
 <tool id="geo_cov_template" name="Make geographic coverage template" version="0.1.0+galaxy0" profile="22.05">
     <description>for EML metadata creation</description>
-    <requirements>
+     <requirements>
```

Comment:
this is not using the container

Reply:
Yes, because there is no change here and it appears there is an issue with this function on the container. Maybe the best is to remove this tool from the PR?

Reply:
But there is better descriptions so it seems OK to keep this tool on the PR but maybe not improve version